Lib/symtable.py

```diff
@@ -221,10 +221,25 @@ def get_methods(self):
         """
         if self.__methods is None:
             d = {}
```

Comment:
Can we use a set here instead? I feel this code may predate the addition of sets to the language.

Reply:
With a set, I would lose the order of discovery, so I think a dict is still good.